src/main/java/net/tardis/mod/common/blocks/BlockGallifreyDirt.java

```diff
@@ -22,24 +22,28 @@
 
 public class BlockGallifreyDirt extends Block implements IGrowable {
 
-    public BlockGallifreyDirt(Boolean isTop) {
+    public BlockGallifreyDirt(Boolean isTop, Boolean hasSnow) {
```

Comment:
BlockStates, please

Reply:
No idea how

Reply:
I will sort this then, but you are fixing the rest